en_ewt-ud-train.conllu

```diff
@@ -164935,8 +164935,8 @@
 # sent_id = answers-20111107144339AA0qw5S_ans-0013
 # text = I my self am one but appreciate that the organization is unlikely to make into the 22nd century.
 1	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	5	nsubj	5:nsubj|7:nsubj	_
-2	my	myself	PRON	GW	Typo=Yes	1	appos	1:appos	_
-3	self	_	X	PRP	_	2	goeswith	2:goeswith	_
+2	my	myself	PRON	PRP	Case=Acc|Number=Sing|Person=1|PronType=Emp|Reflex=Yes|Typo=Yes	1	appos	1:appos	_
+3	self	_	X	GW	_	2	goeswith	2:goeswith	_
```

Comment:
I have a feeling we should leave the XPOS alone for split/goeswith items, even though PRP is on the "wrong" token according to UD, because that is what the Penn standard calls for. Unless @amir-zeldes says otherwise

Reply:
I've now reverted these changes. Thanks.

Reply:
Agreed, conflicting with PTB tokenization/xpos just causes problems and makes datasets into idiosyncratic dialects IMO.